client/engine/chainservice/simulated_backend_chainservice.go

```diff
@@ -57,10 +58,30 @@ type SimulatedBackendChainService struct {
 	sim SimulatedChain
 }
 
+// NewSimulatedBackendChainService constructs a chain service that submits transactions to a NitroAdjudicator
+// and listens to events from an eventSource
+func NewPollingSimulatedBackendChainService(sim SimulatedChain, bindings Bindings,
+	txSigner *bind.TransactOpts, logDestination io.Writer) (ChainService, error) {
+
+	ethChainService, err := newPollingSimulatedChainService(sim,
+		bindings.Adjudicator.Contract,
+		bindings.Adjudicator.Address,
+		bindings.ConsensusApp.Address,
+		bindings.VirtualPaymentApp.Address,
+		txSigner,
+		logDestination)
+	if err != nil {
+		return &SimulatedBackendChainService{}, err
+	}
```

Comment:
I could be missing something, but it doesn't seem terribly worth it to have this wrapper constructor? Could we combine the two?

Reply:
I've merged `newPollingSimulatedChainService` into `NewPollingSimulatedBackendChainService` and renamed `newPollingSimulatedBackendChainService` to keep it private (since we only use it on our tests).